packages/app/src/game/module/chat/sagas.ts

```diff
@@ -0,0 +1,32 @@
+import { take, delay, select, put } from "redux-saga/effects";
+import { quickChatCommands } from ".";
```

Comment:
Importing from your own index file will typically cause problems

When you notice that the current file is even closely related (in the directory tree) to the one you're importing, it's best to be as relative as possible and avoid importing from the index files at all. Prefer to use index files when you're importing something from another "module" or even another library at all

Here, I would try and import `{ setPlayerChat }` directly from the slice